aws-sources-operator/bundle/0.1.0/aws-sources-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -13,31 +13,31 @@ metadata:
           "spec": {
             "adapters": {
               "awscodecommit": {
-                "image": "gcr.io/triggermesh/awscodecommitsource:v0.4.2"
+                "image": "registry.connect.redhat.com/triggermesh/awscodecommitsource:v0.4.2"
               },
               "awscognitoidentity": {
-                "image": "gcr.io/triggermesh/awscognitoidentitysource:v0.4.2"
+                "image": "registry.connect.redhat.com/triggermesh/awscognitoidentitysource:v0.4.2"
               },
               "awscognitouserpool": {
-                "image": "gcr.io/triggermesh/awscognitouserpoolsource:v0.4.2"
+                "image": "registry.connect.redhat.com/triggermesh/awscognitouserpoolsource:v0.4.2"
               },
               "awsdynamodb": {
-                "image": "gcr.io/triggermesh/awsdynamodbsource:v0.4.2"
+                "image": "registry.connect.redhat.com/triggermesh/awsdynamodbsource:v0.4.2"
               },
               "awskinesis": {
-                "image": "gcr.io/triggermesh/awskinesissource:v0.4.2"
+                "image": "registry.connect.redhat.com/triggermesh/awskinesissource:v0.4.2"
               },
               "awssns": {
-                "image": "gcr.io/triggermesh/awssnssource:v0.4.2"
+                "image": "registry.connect.redhat.com/triggermesh/awssnssource:v0.4.2"
```

Comment:
This one will probably not work if we release a new operator without bumping the images, because of the changes introduced in #187. One solution would be to revert only the changes to the chart, and remember to re-apply them on the next release.

Reply:
the operator only contains the helm chart. in theory we only need to build the operator again of the chart has updates (manifests added / modified). the images to be deployed by the operator are actually listed here in the clusterserviceversion (essentially it's the values.yaml for our chart).

Reply:
That's the point, `v0.4.2` still requires the KNATIVE_URL_SCHEME and KNATIVE_DOMAIN environment variables, but those are gone since #187, so `v0.4.2` won't work with the Helm chart @ `master`.

Reply:
the version that will get installed now is https://github.com/triggermesh/aws-event-sources/blob/master/aws-sources-operator/bundle/0.2.0/aws-sources-operator.v0.2.0.clusterserviceversion.yaml. it uses the 0.5.0 images.